aiida/orm/querybuilder.py

```diff
@@ -55,6 +60,28 @@
 # subclassing for any entity type. This workaround should then be able to be removed.
 GROUP_ENTITY_TYPE_PREFIX = 'group.'
 
+# re-usable type annotations
+NODE_CLS_TYPE = Type[Any]  # pylint: disable=invalid-name
+PROJECT_TYPE = Union[str, dict, Sequence[Union[str, dict]]]  # pylint: disable=invalid-name
+FILTER_TYPE = Dict[str, Any]  # pylint: disable=invalid-name
+ROW_TYPE = Any  # pylint: disable=invalid-name
```

Comment:
Don't know really lol, I think I say it somewhere else and kind of liked it, to distinguish these type annotation variables from regular one. But its not a big deal, so fine to change

Reply:
I just find it confusing when reading the type annotations seeing `PathItemType` and then `ROW_TYPE`. Think if that is also `RowType` it would be clearer and finding where the definition comes from is easy

Reply:
no worries, changed it 👍